php/php.editor/src/org/netbeans/modules/php/editor/indent/TokenFormatter.java

```diff
@@ -3211,53 +3244,71 @@ private int countSpacesForGroupedToken(final FormatToken.AssignmentAnchorToken t
 
         private int countSpacesWhenNotExpandingTabs(final FormatToken.AssignmentAnchorToken token) {
             int spaces;
+            int spaceAroundAssignment = getSpaceAroundAssignment(token);
+            // consider also the space around assinment here
+            // to avoid adding extra spaces before assignment
```

Comment:
I've changed this method behavior.

e.g. See below unit tests:
- https://github.com/apache/netbeans/pull/6884/files#diff-bcbc4870ac3ca73b9a4a6ff60602fc997de403e1ff13f370a5652d9a340f96fb
- https://github.com/apache/netbeans/pull/6884/files#diff-08ebd15b297e731302f5c9374c2f57687a0d58f0b41b1cdca4ac40b014931742

Before:

![nb-php-gh-6074-match-arm-arrow-alignment-extra-space-before](https://github.com/apache/netbeans/assets/738383/9faeb797-2640-4c3a-a2f5-bb2c11ee1502)

After:

![nb-php-gh-6074-match-arm-arrow-alignment-extra-space-after](https://github.com/apache/netbeans/assets/738383/02b87432-ff31-45d2-ba56-6cc828991a02)

Reply:
Looks correct to me now 👍